app/helpers/application_helper.rb

```diff
@@ -1,2 +1,10 @@
 module ApplicationHelper
+    def full_title(page_title = '')
+        base_title = "Ruby on Rails Tutorial Sample App"
```

Comment:
If you are feeling lucky perhaps localise this copy? I did a very old version of the tutorial years ago so I cant remember if localising copy is included. 

We localise (or are supposed to) all the text in DocFox to multiple distinct languages (all English but us and sa are different and banking uses different terms or other financial business)